persistent/ChangeLog.md

```diff
@@ -2,13 +2,18 @@
 
 ## unreleased
 
-* [#1460] https://github.com/yesodweb/persistent/pull/1468
+* [#1460](https://github.com/yesodweb/persistent/pull/1468)
     * Remove extraneous `map toPersistValue` call in the `mkInsertValues`
       function, as it evaluates to `id`.
 
+## 2.14.5.0
+
+* [#1437](https://github.com/yesodweb/persistent/pull/1437)
+    * Add `existsBy` to `PersistUniqueRead`
+
 ## 2.14.4.4
 
-* [#1460] https://github.com/yesodweb/persistent/pull/1460
+* [#1460](https://github.com/yesodweb/persistent/pull/1460)
```

Comment:
This matches the format in the rest of the changelog, where the PR number links to the link, but the URL itself is not shown. I can revert these changes if you want.

Reply:
That's great - thank you!